### ISSUE TITLE
NTR: Bone marrow: lipid-laden macrophages (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9175]
New term request Bone marrow: lipid-laden macrophages (elthat_100822111116): Foam cells  UMLS: 1845014

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000640
- label: Lipid-laden bone-marrow macrophages
- parents: [Abnormal bone marrow cell morphology (HP:0005561)](https://hpo.jax.org/app/browse/term/HP:0005561)
- pmid: 
- def: Bone marrow with increased numbers of macrophages heavily laden with lipids.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9175
- synonyms:
